### DISCUSSION TITLE
Cannot see any comment even as admin

### DISCUSSION THREAD

**Discussion by rokdd** [directus/directus#14722] [Archive]
I created a comment by admin account, and they are saved (no error in UI and log) in the database (row visible). But in the UI when I view the item it does not show the comments. The comment is listed in the activitys.

![image](https://user-images.githubusercontent.com/640866/181529945-a42aa529-cc35-4e8f-a605-aa9dc61d8b2e.png)

**Comment by azrikahar** [comment]
If you're on Directus `9.14.2` and above, it should be due to the known bug #14608. (where it's retrieving the wrong comment. You should be able to see the activity API call in network tab fetching by `+` instead of the correct ID)

**Reply by rokdd** [reply]
Thanks that seems the same problem.. I searched for "comment" I could not find this bug. However it is only in viewing so far I understand and hopefully it is fixed soon!
